lib/java/com/google/android/material/lists/res/values/dimens.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!--General dimensions-->
+    <dimen name="material_list_item_side_padding">16dp</dimen>
+    <dimen name="material_list_item_item_padding">16dp</dimen>
+    <dimen name="material_list_item_medium_visual_size">40dp</dimen>
+    <dimen name="material_list_item_meta_border">28dp</dimen>
+    <dimen name="material_list_item_large_item_padding">32dp</dimen>
+    <dimen name="material_list_item_action_size">24dp</dimen>
+    <dimen name="material_list_item_rectangle_width">100dp</dimen>
+    <dimen name="material_list_item_rectangle_height">56dp</dimen>
+    <dimen name="material_list_item_square_size">40dp</dimen>
+    <dimen name="material_list_item_meta_height">32dp</dimen>
+    <!-- The meta text uses large padding instead of the recommended 28dp to have the meta text better line
+   up with the primary texts in the "two_line_item" and "three_line_item" xml layouts. -->
+
+
+   <!-- Single-line specific dimensions -->
+    <dimen name="material_list_item_single_line_medium_height">56dp</dimen>
+    <dimen name="material_list_item_single_line_small_height">48dp</dimen>
+    <dimen name="material_list_item_single_line_large_height">72dp</dimen>
+
+    <!-- Two-line specific dimensions -->
+    <dimen name="material_list_item_two_line_large_height">72dp</dimen>
+    <dimen name="material_list_item_two_line_small_height">64dp</dimen>
+    <dimen name="material_list_item_two_line_overline_height">24dp</dimen>
+    <dimen name="material_list_item_two_line_primary_text_height">20dp</dimen>
+
+    <!-- Three-line specific dimensions -->
+    <dimen name="material_list_item_three_line_height">88dp</dimen>
+    <dimen name="material_list_item_large_square_size">56dp</dimen>
+    <dimen name="material_list_item_three_line_overline_height">28dp</dimen>
+    <dimen name="material_list_item_three_line_rectangle_end_margin">20dp</dimen>
```

Comment:
I suggest that each of the dimensions should be documented with a comment.

As for the comments separating the dimensions, you should use `<eat-comment />` below the comments such that they won't be used for documenting the dimensions.